src/components/Content/Content.styled.ts

```diff
@@ -10,13 +10,33 @@ export const MainWrapper = styled.div`
 	&.content-wrapper + .content-wrapper {
 		border-top: 2px solid ${theme.newColors.grey2["100"]};
 	}
+
+	&.card-wrapper {
+		border: 2px solid ${theme.newColors.grey2["100"]};
+		width: 100%;
+		padding: 0px;
+		margin: 0px;
+	}
+
+	& .content-wrap {
```

Comment:
Now that the HR is going away, there's no need for this class. The padding could go directly in the ContentRow component using the class I suggested in Content.tsx